admin_guide/managing_networking.adoc

```diff
@@ -275,6 +275,12 @@ above example, suppose `www.foo.com` resolved to `10.11.12.13` and has a DNS TTL
 of one minute, but was later changed to `20.21.22.23`. {product-title} will then
 take up to one minute to adapt to these changes.
 
+[NOTE]
+====
+If you don't use the default install and you include a deny all rule you may need to explicitly allow traffic to your DNS server.
```

Comment:
I'd like this to be expanded a little further to explain why the default install works (i.e. we should say that we always allow access to the external interface of the host the pod is on).  We should also say that they need to allow access to the DNS servers (not the plural) by IP address (just to avoid stupidity).